backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/steps/Map.kt

```diff
@@ -44,8 +44,12 @@ class Map(private val logger: Logger) : PipelineStep<List<LbeAcceptingStore>, Li
         return if (int == ALTERNATIVE_MISCELLANEOUS_CATEGORY) MISCELLANEOUS_CATEGORY else int
     }
 
-    private fun String?.clean(): String? {
-        return this?.replaceNa()?.trim()
+    private fun String?.clean(removeSubsequentSpaces: Boolean = true): String? {
+        val trimmed = this?.replaceNa()?.trim()
+        if (removeSubsequentSpaces) {
+            return trimmed?.replace(Regex(""" {2,}"""), " ")
```

Comment:
Maybe we want to match for any whitespace (including newlines) as well here?
See "\s" here: https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html

Reply:
I first had `\s` but that broke the content of some of the discount fields with longer text since we just replaced newlines it with a single space which makes it less readable. This is currently not a problem since we don't replace spaces in the `discount` field anyway so I am open to changing it.

Reply:
`\s` makes sense for the name of a Akzeptanzstelle. I don't think it makes sense for the discount. I think we should not touch the discount until we observe issues. `replaceNa` and `trim` should be fine for discount though

Reply:
I think the @LanuageRegex thingy is missing here :)

Reply:
Will change to `\s` :+1: 
The @LanguageRegex thing is only to tell IntelliJ it is a regex for highlighting/tools and is only needed for plain strings. In this case, IntelliJ detects automatically that is a regex because we use the `Regex` class.

Reply:
I see! However, any newlines or tabs (or any whitespace that is not a single space) should be removed from all other fields except discount anyways I guess 🤔